docs/source/concepts/FittingMinimizers.rst

```diff
@@ -0,0 +1,205 @@
+terms of how they perform when fitting several benchmark
+problems. This is a relative comparison in the sense that for every
+problem the best possible results with Mantid minimizers are given a
+top score of "1". The ranking is continuous and the score of a
+minimizer represents the ratio between its performance and the
+performance of the best. We compare accuracy and run time.
+
+.. Q: accuracy metric: sum of squared residuals or chi2 variants
+
+For example, a ranking of 1.25 for a minimizer for a given problem
+means:
+
+- Referring to the accuracy of a minimizer, it produces a solution
+  with squared residuals 25% larger than the best solution in Mantid.
+- Referring to the run time, it takes 25% more time than the fastest
+  minimizer.
+
+All the minimizers available in Mantid 3.7 were compared, with the
+exception of FABADA which belongs to a different class of methods and
+would not be compared in a fair manner. Minimizers are used here as
+"black boxes", without any special initialization, constraints, or
+rewriting of the equations to fit.
+
+Accuracy is measured using the sum of squared fitting errors as
+metric, or "ChiSquared" as defined in :ref:`Fit
+<algm-CalculateChiSquared>`, where the fitting errors are the
+difference between the expected outputs and the outputs calculated by
+the model fitted: :math:`\chi_{1}^{2} = \sum_{i} (y_i - f_i)^2` (see
+:ref:`CalculateChiSquared <algm-CalculateChiSquared>` for full details
+and different variants).  Run time is measured for the execution of
+the :ref:`Fit <algm-Fit>` algorithm for an equation and dataset
+previously created.
+
+.. There would be two alternative for the errors:
+   1. Without errors, as it is
+   2. With "simulated" sqrt(X) errors
+   Here we use 2 because Levenberg-Marquardt doesn't work with 1 ('Unweighted least squares' cost function
+
+The cost function used in all cases is `Least squares` (weighted by
+the errors, as usually applied in Mantid).
+
+
+Benchmark problems
+##################
+
+Each test problem included in this comparison is defined by the
+following information:
+
+- Input data (:math:`x_i` values)
+- Output data (:math:`y_i` values)
+- Function to fit, with parameters
+- Starting point or initial values of the function parameters
+- Certified or reference best values for the parameters, with an associated residual of the certified or best model 
+
+The problems have been obtained from two different benchmarks:
+
+- `NIST nonlinear regression problems <http://itl.nist.gov/div898/strd/general/dataarchive.html>`__.
+
+- The `Constrained and Unconstrained Testing Environment <http://www.cuter.rl.ac.uk/Problems/mastsif.html>`__
+  set of test problems for linear and nonlinear optimization.
+
+As the NIST problems do not include measurement errors, assuming that
+these datasets would represent spectra from a typical Mantid workspace
+we introduce errors as the square root of the observations.
```

Comment:
I'll try that. You mean setting all errors to zero and set the flag to ignore them all? We'd still have to see what do do about this.
Not using errors is good in that it keeps the original NIST problems, but it is bad in that it goes further in the direction of having "toy" problems that do not resemble our users' problems.
Yes, the chi squared is calculated directly from the residuals.

 If it works well without errors we could show that for simplicity, but I'd then at least mention somewhere how it changes with fake errors, or give the tables obtained with errors in yet one more subpage.